buildrunner/cli.py

```diff
@@ -228,11 +228,11 @@ def parse_args(argv):
     )
 
     parser.add_argument(
-        '--single-platform',
-        default=False,
-        action='store_true',
-        dest='single_platform',
-        help="overrides the 'platforms' configuration and only builds a single platform image",
+        '--disable-multi-platform',
+        default=None,
+        choices=['True', 'true', 'False', 'false'],
```

Comment:
why not just make it `true` or `false` and forget about the more pythonic form since it's on the command line anyways?